controller/productController.js

```diff
@@ -0,0 +1,81 @@
+const productService = require('../services/productService')
+const middleJwt = require('../middleware/jwt');
+
+const productAll = async(req,res) =>{
+    try{
+        const {brandName,scentName} = req.query
+        const products = await productService.getProducts(brandName,scentName)
+        res.status(200).json({products})
+    }catch(err){
+        res.status(500 || statusCode).json({message: err.message})
+    }
+}
+
+const getProduct = async(req,res) => {
+    const id = req.params.id
+    try{
+        const product = await productService.getProduct(id)
+        res.status(200).json({product})
+    }catch(err){
+        res.status(err.statusCode).json({message: err.message})
+    }
+}
+const createProduct = async(req,res) =>{
+    const {productName,price,description,brandId,scentId,imageId} = req.body
+    const token = req.headers.authorization.substr(7)
+    const decodedToken = await middleJwt.verifyToken(token)
+    try{
+        if(!productName||!price||!description||!brandId||!scentId||!imageId){
+            throw new Error("INPUT_KEY_ERROR")
+        }
+        if(token){
+            if(decodedToken.status === 1){
+                await productService.createProduct(productName,price,description,brandId,scentId,imageId)
+                res.status(200).json({message: "successfully created"})
+            }else{
+                throw new Error("Permission Denied")
+            }
+        }
+    }catch(err){
+        res.status(404).json({message: err.message})
+    }
+}
+const updateProduct = async (req,res) =>{
+    const productId = req.params.productId
+    const {productName,price,description,brandId,scentId,imageId} = req.body
+    const token = req.headers.authorization.substr(7)
+    const decodedToken = await middleJwt.verifyToken(token)
+    try{
+        if(token){
+            if(decodedToken.status === 1){
+                await productService.updateProduct(productId,productName,price,description,brandId,scentId,imageId)
+                res.status(200).json({message: "successfully updated"})
+            }else{
+                throw new Error("Permission Denied")
+            }
+        }
+    }catch(err){
```

Comment:
```suggestion
        if(token){
            const userType = {
              ADMIN : 1
            }
            if(decodedToken.status === userType.ADMIN){
                await productService.updateProduct(productId,productName,price,description,brandId,scentId,imageId)
                res.status(200).json({message: "successfully updated"})
            }else{
                throw new Error("Permission Denied")
            }
        }
    }catch(err){
```
1. 에러 statusCode가 없어요, 추가해 주세요 :)
2. decodedToken.status 1번이 어떤 상태인지 판단하기가 어렵습니다. 상위에 객체 하나 선언하셔서 어떤 상태인지 보다 명확히 해주세요